datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/testable.py

```diff
@@ -129,9 +129,11 @@ def _get_and_parse_commits(base_ref: str, target_ref: str) -> List[Tuple[str, st
     lines: List[str] = result.stdout.splitlines()
 
     commits = []
-
     for line in reversed(lines):
-        commit_hash, _, commit_subject = line.partition(' ')
+        sign, _, commit = line.partition(' ')
+        if sign == '-':
+            continue
+        commit_hash, _, commit_subject = commit.partition(' ')
```

Comment:
```suggestion
        sign, commit_hash, commit_subject = line.split(' ', 2)
        if sign == '-':
            continue
```

We can just split once like above ?